Sources/MediaNavigationBar.swift

```diff
@@ -0,0 +1,128 @@
+/*****************************************************************************
+ * MediaNavigationBar.swift
+ *
+ * Copyright © 2019 VLC authors and VideoLAN
+ *
+ * Authors: Robert Gordon <robwaynegordon # gmail.com>
+ *
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+@objc (VLCMediaNavigationBarDelegate)
+protocol MediaNavigationBarDelegate {
+    func didMinimizePlayback(_ navigationBar: MediaNavigationBar)
+    func didToggleChromeCast(_ navigationBar: MediaNavigationBar)
+    func didToggleAirPlay(_ navigationBar: MediaNavigationBar)
+}
+
+@objc (VLCMediaNavigationBar)
+@objcMembers class MediaNavigationBar: UIView {
+    
+    // MARK: Instance Variables
+    weak var delegate: MediaNavigationBarDelegate?
+    
+    var hideTitleLabel: Bool {
+        didSet {
+            mediaTitleTextLabel.isHidden = hideTitleLabel
+        }
+    }
+    
+    lazy var minimizePlaybackButton: UIButton = {
+        var minButton = UIButton(type: .system)
+        minButton.addTarget(self, action: #selector(minimizePlayback), for: .touchUpInside)
+        minButton.setImage(UIImage(named: "iconChevron"), for: .normal)
+        minButton.tintColor = .white
+        minButton.translatesAutoresizingMaskIntoConstraints = false
+        return minButton
+    }()
+    
+    private var mediaTitleTextLabel: UILabel = {
+        var label = UILabel()
+        label.numberOfLines = 1
+        label.textColor = .white
+        label.font = UIFont(name: "SFProDisplay-Medium", size: 17)
```

Comment:
systemfont. the systemfont is sfprodisplay by default but by using that one we will have no trouble should apple change it in the future to something else